tests/unit/jupyter/test_jupyter_input_generation.py

```diff
@@ -36,8 +34,8 @@ async def test_jupyter_values_generation(setup_clients):
         app_secrets_name=APP_SECRETS_NAME,
     )
     assert helm_params["image"] == {
-        "repository": "ghcr.io/neuro-inc/base",
-        "tag": "v25.3.0-runtime",
+        "repository": "quay.io/jupyter/base-notebook",
```

Comment:
what if we expose selector for the user with our base images, those base images & raw string field if user wants to specify own list of images?

Also, we could ease the problem of long pulling the image by updating cluster configuration to prepull our images on each node. wdyt? this is already there, the problem is that list was not updated for a long time.
```
imagesPrepull:                                                                                           
  images:                                                                                                
  - image: neuromation/base                                                                              
  - image: neuromation/web-shell 
 ```